XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1851,6 +1851,31 @@
             "TextBox"
           ]
         },
+        {
+          "UniqueId": "NumberBox",
+          "Title": "NumberBox",
+          "Subtitle": "A single-line or multi-line plain text field.",
+          "ImagePath": "ms-appx:///Assets/NumberBox.png",
+          "Description": "Use a NumberBox to let a user enter numeric and formulaic input in your app.",
```

Comment:
I may be overthinking, but "formulaic input" doesn't sound right to me.

Reply:
I mean... it does support formulaic input, right?

Reply:
definition of formulaic: "constituting or containing a verbal formula or set form of words; produced in accordance with a slavishly followed rule or style; predictable."

A better description might be: Use NumberBox to allow users to enter numeric input and evaluate algebraic  equations in your app.